gameci-1_intro.md

```diff
@@ -0,0 +1,146 @@
+# GameCI 1: Intro to GitHub Actions for Unity
+
+As a software engineer, I believe every modern software project should have a CI/CD pipeline.
+The reasons for such are many and better explained [all](https://www.synopsys.com/glossary/what-is-cicd.html)
+ [over](https://www.redhat.com/en/topics/devops/what-is-ci-cd) [the](https://www.infoworld.com/article/3271126/what-is-cicd-continuous-integration-and-continuous-delivery-explained.html)
+ [internet](https://en.wikipedia.org/wiki/CI/CD), so I won't be making a case for why you should be using CI/CD.
+Instead, I write this series of articles with the goal of helping Game Developers with their own pipelines.
+In particular, I believe that [Unity](https://unity.com/) game projects don't have as many resources for CI/CD as they should.
+Hopefully, this guide to how I built the CI/CD pipeline for my Unity project will help you with yours.
+
+## My Workflow
+
+A picture is worth a thousand words, so take a look at the visualization graph for my workflow:
+![Test, Build, and Deploy with GameCI](assets/img/cgs-workflow.png)
+
+Nowadays, developers have a lot of CI options (Unity Cloud Build, CircleCI, GitLab CI, and Jenkins are just a few examples).
+I chose [GitHub Actions](https://github.com/features/actions) because it is tightly integrated to the GitHub repo where I already keep my open-source project,
+ and GitHub provides free CI minutes for open-source projects like mine.
+Plus, it makes this nice visualization graph.
```

Comment:
```suggestion
I chose [GitHub Actions](https://github.com/features/actions) because it is tightly integrated with the GitHub repository where I already keep my open-source project
 and GitHub provides free CI minutes for open-source projects.
Plus, it makes this nice visual graph.
```